features/cellular/framework/API/CellularSIM.h

```diff
@@ -18,12 +18,16 @@
 #ifndef CELLULAR_SIM_H_
 #define CELLULAR_SIM_H_
 
+#include <stddef.h>
```

Comment:
how does this relates to the new API addition?

Reply:
That's for size_t definition.